### PR TITLE
TINY-7330: Added manifest files for Edge and Google Chrome 90

### PR DESCRIPTION
**Pull request by lnewson**

### CODE REVIEW

File: googlechrome-90.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            "url": "https://dl.google.com/release2/chrome/VYWR7z03VITSenbdO3GEAQ_90.0.4430.72/90.0.4430.72_chrome_installer.exe#/dl.7z",
+            "hash": "704a58c4ccb49d82eea8c6c8354621c9c481fdb0bbc3a070c3d22697475a6332"
+        },
+        "32bit": {
+            "url": "https://dl.google.com/release2/chrome/QXfLrKD3khekJ96AJ562-g_90.0.4430.72/90.0.4430.72_chrome_installer.exe#/dl.7z",
+            "hash": "d29a4c3b994a1f14fec3a67bb44789fe6d44fb0dcc7b7a4a0239f2fa54abadc5"
+        }
+    },
+    "installer": {
+        "script": "Expand-7zipArchive \"$dir\\chrome.7z\" -ExtractDir 'Chrome-bin' -Removal"
```
Comment (ashfordneil) [inline googlechrome-90.json:20]: How come this needs an installer step, when the chrome driver and edge driver manifests just said "download the zip, and the binary name is X"?

File: googlechrome-90.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "bin": "chrome.exe",
+    "shortcuts": [
+        [
+            "chrome.exe",
+            "Google Chrome"
+        ]
+    ],
+    "checkver": {
+        "url": "https://chrome-dl.com/api/chrome.min.xml",
+        "regex": "(?sm)<stable32><version>(?<version>[\\d.]+)</version>.+release2/chrome/(?<32>[\\w-]+)_.+<stable64>.+release2/chrome/(?<64>[\\w-]+)_.+</stable64>"
```
Comment (ashfordneil) [inline googlechrome-90.json:31]: If this is a hand-written regex, let me know and I'll spend the time reviewing it. If it's been downloaded from an upstream then I'll ignore

**Review by ashfordneil — APPROVED** [review]

File: googlechrome-90.json (hunk 1 above)
Comment (lnewson) [reply googlechrome-90.json:20]: This is all just boilerplate from the original version. It's also in all the other versions: https://github.com/tinymce/scoop-bucket/blob/master/googlechrome-81.json#L20

Also don't forget this is Google Chrome itself, not a webdriver.

File: googlechrome-90.json (hunk 2 above)
Comment (lnewson) [reply googlechrome-90.json:31]: Yeah, it's just copied from the original version that's always using `latest` as I understand it. So this is just a copy of the old version with the version, shas and download path updated.

**Review by metricjs — APPROVED** [review]

**Pull request merged**